Ms get currencies

MSALCEDOH opened the pull request:
Adding get currencies method.
Enabling swagger

MSALCEDOH committed:
Enabling swagger

MSALCEDOH committed:
Adding get Currency method

lbricenob commented on Budget.Web.App/Controllers/CurrencyController.cs:
```diff
@@ -0,0 +1,18 @@
+        [HttpGet]
+        public IActionResult GetCurrency()
+        {
+            var json = System.IO.File.ReadAllText(@"Files/Currencies.json");
```
no uses "var", en todo caso "string"

lbricenob commented on Budget.Web.App/Controllers/CurrencyController.cs:
```diff
@@ -0,0 +1,18 @@
+        public IActionResult GetCurrency()
+        {
+            var json = System.IO.File.ReadAllText(@"Files/Currencies.json");
+            return Content(json, "application/json");
```
hay que crear una clase
Currency{
string CodeCurrency,
string DesCurrency
}

y deserializalo a esa clase. La idea es que devuelvas un List<Currency>

MSALCEDOH committed:
Adding Currency entity
Changing method GetCurrency to return a List of Currency
Changing json file to be with a correct format
